pages/DSNP/Identity.md

```diff
@@ -19,13 +19,34 @@ A user's ownership of their identity is expressed via ownership and control of t
 Control entails the power to announce content associated with the identifier and
 the ability to delegate permission to others to announce content on the user's behalf.
 
+## Retirement
+
+A user may choose to retire their identity at any time.
+Once an identifier is retired, an implementation MAY remove all state data associated with that identifier, provided that an indication that the identifier is retired remains, so it may not be reused in the future.
+This means that all data previously sent from the identifier, the control keys associated with the identifier, and the delegations (see next section) associated with the identifier may be removed.
+
 ### Delegation
```

Comment:
This was originally a sub heading of `## Ownership`. So we either want to move Retirement below it, or raise Delegation to `##` instead of `###`.

Reply:
Thanks, will fix.